### PR TITLE
add labels to tenant secrets for easy deletion

### PR DESCRIPTION
**Pull request by Alevsk**

### CODE REVIEW

File: restapi/admin_tenants.go [hunk 1]
```diff
@@ -769,6 +782,7 @@ func setImageRegistry(ctx context.Context, tenantName string, req *models.ImageR
 		},
 		Data: map[string][]byte{
 			corev1.DockerConfigJsonKey: []byte(string(imRegistryJSON)),
+			operator.TenantLabel:       []byte(tenantName),
```
Comment (cesnietor) [inline restapi/admin_tenants.go:785]: shuldn't this be as label?
```
Labels: map[string]string{
						operator.TenantLabel: tenantName,
					},
```

Comment (cesnietor) [reply restapi/admin_tenants.go:785]: because if it doesnt exist we create it wih this content

Comment (dvaldivia) [reply restapi/admin_tenants.go:785]: does this needs to be in the data of the secret??

Comment (Alevsk) [reply restapi/admin_tenants.go]: good catch, yes this should be in labels field

**Pull request merged**